src/utils/translate/fr.json

```diff
@@ -554,6 +552,15 @@
   "Today": "Aujourd'hui",
   "TotalEstimates": "Estimations totales",
   "TotalTests": "Total des tests",
+  "UnityBanner": {
+    "Currently": "En ce moment, seulement les études alignées sur le ",
+    "ProtocolLink": "protocole séro-épidémiologique de l'OMS Unity ",
+    "ViewDatabase": "sont affichées. Pour afficher notre base de données complète, décochez le filtre",
+    "WHOUnityFilter": "seulement OMS Unity.",
+    "Filter": ""

```

Comment:
missed this

Reply:
According to the translation, there comes something after the who unity link in english, but in french it ends with the link, so I had to leave it as an empty string